benchmark/metrics.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Benchmark
+  module Metrics
+    # put metrics generated from `run_benchmarks` to cloudwatch
+    def self.put_metric(client, dims, timestamp, k, v)
```

Comment:
Positional args for this is kind of weird .. I know it's like this in v3

Reply:
Updated to kwargs